scipy/stats/qmc.py

```diff
@@ -1361,7 +1420,7 @@ class MultivariateNormalQMC(QMCEngine):
 
     """
 
-    def __init__(self, mean, cov, inv_transform=False, engine=None, seed=None):
+    def __init__(self, mean, cov, inv_transform=True, engine=None, seed=None):
```

Comment:
Probably want to do this in the constructor of `NormalQMC` as well? One potential issue with this is that for some extremely low/high draws the inverse cdf mapping could result in numerical issues / spit out nans (not sure if this is an issue with the scipy implementation). 

one potential concern with this default is that

Reply:
Indeed, I forgot `NormalQMC`.

As for inverse over Box-Muller, I changed it because the others pointed out that we should use the inverse with Sobol'.